pallets/automation-time/src/lib.rs

```diff
@@ -1344,19 +1355,34 @@ pub mod pallet {
 			.map_err(|_| Error::<T>::TimeSlotFull)
 		}
 
+		// TODO: move to a utils upon this MVP works
+		// We cannot use Rust std inside substrate runtime so i need to re-invent this
+		// Convert a decimal number to its string re-presentation where each character is a digit
+		// of the original number
+		pub fn u64tovec(a: u64) -> Vec<u8> {
+			let mut s = Vec::new();
+			let mut b: u64 = a;
+			loop {
+				let digit = b % 10;
+				b = (b - digit) / 10;
+				// 49 -> 0
+				s.push((48 + digit) as u8);
+				if b <= 0 {
+					break
+				}
+			}
+			s.reverse();
+			s
+		}
+
 		/// TODO ENG-538: Refactor validate_and_schedule_task function
```

Comment:
ENG-538: Refactor validate_and_schedule_task function is deleted on Jira. Could you help remove thes lines of function comment?